tmt/schemas/tests.yaml

```
@@ -0,0 +1,129 @@
+---
+
+#
+# JSON Schema definition for tmt `Tests`
+#
+# https://tmt.readthedocs.io/en/stable/spec/tests.html
+#
+
+$id: /schemas/test
+$schema: https://json-schema.org/draft-07/schema
+
+type: object
+additionalProperties: false
+
+properties:
+
+  # https://tmt.readthedocs.io/en/stable/spec/core.html#adjust
+  adjust:
+    $ref: "/schemas/core#/definitions/adjust"
+
+  # https://tmt.readthedocs.io/en/stable/spec/tests.html#component
+  component:
+    $ref: "/schemas/common#/definitions/string_or_array"
+
+  # https://tmt.readthedocs.io/en/stable/spec/tests.html#contact
+  contact:
+    $ref: "/schemas/common#/definitions/string_or_array"
+
+  # https://tmt.readthedocs.io/en/stable/spec/core.html#description
+  description:
+    $ref: "/schemas/core#/definitions/description"
+
+  # https://tmt.readthedocs.io/en/stable/spec/tests.html#duration
+  duration:
+    type: string
+    pattern: "^[0-9]+[smhd]$"
+
+  # https://tmt.readthedocs.io/en/stable/spec/core.html#enabled
+  enabled:
+    $ref: "/schemas/core#/definitions/enabled"
+
+  # https://tmt.readthedocs.io/en/stable/spec/tests.html#environment
+  environment:
+    type: object
+
+    additionalProperties:
+      type: string
+
+  # https://tmt.readthedocs.io/en/stable/spec/tests.html#framework
+  framework:
+    type: string
+
+    enum:
+      - beakerlib
+      - shell
+
+  # https://tmt.readthedocs.io/en/stable/spec/core.html#link
+  link:
+    $ref: "/schemas/core#/definitions/link"
+
+  # https://tmt.readthedocs.io/en/stable/spec/tests.html#manual
+  manual:
+    type: boolean
+
+  # https://tmt.readthedocs.io/en/stable/spec/core.html#order
+  order:
+    $ref: "/schemas/core#/definitions/order"
+
+  # https://tmt.readthedocs.io/en/stable/spec/tests.html#path
+  path:
+    type: string
+    pattern: "^/.+"
```

```suggestion
    pattern: "^/.*"
```

Root path `path: /` is also valid.

---

ack, ty

---

actually just `^/` should be enough, it is the same thing